OrbitGl/SamplingReport.h

```diff
@@ -50,7 +49,7 @@ class SamplingReport {
   std::vector<SamplingReportDataView> thread_reports_;
   CallStackDataView* callstack_data_view_;
 
-  uint64_t selected_address_;
+  std::vector<uint64_t> selected_addresses_;
```

Comment:
Thanks for pointing out this. I changed to use `set` instead of `vector` here.

I'm not sure whether there will be functions with the same address in the sampling view. I suppose there is no duplicates for the displayed addresses are function base addresses. The duplicate addresses will not cause issues for finding callstacks according to selected address. 
But I agree that there is no harm to use a `set`, it can prevent any potential issues if there exists duplicated addresses.